swap_meet/decor.py

```diff
@@ -1,2 +1,10 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+# Wave 5
+#=========================================  
+class Decor(Item):
+    def __init__(self, condition=0):
+        super().__init__(category = "Decor", condition = condition)
+
+    def __str__(self):
+        return "Something to decorate your space."
```

Comment:
Personal hot take: while Python won't have compiling errors if there's no newline at the end of the file, some languages will be more fussy so it's a good practice to have a blank line at the end!